crates/compiler/parse/src/ident.rs

```diff
@@ -398,6 +401,15 @@ fn chomp_accessor(buffer: &[u8], pos: Position) -> Result<Accessor, BadIdent> {
     }
 }
 
+/// a `&foo` or `&1` updater function
+fn chomp_updater(buffer: &[u8], pos: Position) -> Result<&str, BadIdent> {
```

Comment:
```suggestion
/// a `&foo` updater function
fn chomp_updater(buffer: &[u8], pos: Position) -> Result<&str, BadIdent> {
```

since this only applies to lowercase fields